scripts/custom/clean_osiris_codes.py

```diff
@@ -0,0 +1,20 @@
+from scripts import utils
+import re
+
+
+# function clean_osiris_codes
```

Comment:
The good practice for comments is to add them under
`def clean_codes():`
"""Some description here
"""